src/pool/srv_pool.c

```diff
@@ -3583,9 +3605,35 @@ ds_pool_evict_handler(crt_rpc_t *rpc)
 	if (rc != 0)
 		D_GOTO(out_lock, rc);
 
-	if (n_hdl_uuids > 0)
-		rc = pool_disconnect_hdls(&tx, svc, hdl_uuids, n_hdl_uuids,
-					  rpc->cr_ctx);
+	if (n_hdl_uuids > 0) {
+		/* If pool destroy but not forcibly, error: the pool is busy */
+
+		if ((in->pvi_pool_destroy) &&
+		    (in->pvi_pool_destroy_force == 0)) {
```

Comment:
Heheh, these two use contradicting styles... :) Should be

    if (pool_destroy && !pool_destroy_force)

Reply:
addressed in most recent patch